packages/react-components/src/components/loader.tsx

```diff
@@ -1,15 +1,43 @@
+import { keyframes } from '@emotion/core'
+import styled from '@emotion/styled'
+import { giphyBlue, giphyGreen, giphyPurple, giphyRed, giphyYellow } from '@giphy/js-brand'
 import React from 'react'
-import { loader } from '@giphy/js-brand'
-import { cx } from 'emotion'
 
-// trying to share css from brand, but this has an element structure
-// so it's ugly. will probably move the css in here
+const bouncer = keyframes`
+     to {
+    transform: scale(1.75) translateY(-20px);
+  }
+`
+const loaderHeight = 37
+const Loader = styled.div`
+    display: flex;
+    align-items: center;
+    height: ${loaderHeight}px;
+    padding-top: 15px;
+    margin: 0 auto;
+    text-align: center;
+    justify-content: center;
+    animation: pulse 0.8s ease-in-out 0s infinite alternate backwards;
+`
+
+const Dot = styled.div<{ color: string; delay: string }>`
+    display: inline-block;
+    height: 10px;
+    width: 10px;
+    margin: ${loaderHeight}px 10px 10px 10px;
+    position: relative;
+    box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.3);
+    animation: ${bouncer} cubic-bezier(0.455, 0.03, 0.515, 0.955) 0.75s infinite alternate;
+    background: ${(props) => props.color};
+    animation-delay: ${(props) => props.delay};
+`
+
 export default ({ className = '' }: { className?: string }) => (
-    <div className={cx(loader, className)}>
```

Comment:
This was awkward anyway, I was too ambitious trying to have `@giphy/js-brand` export classNames that could be used in any project regardless of that project's framework. Just redid the loader in React and avoid all that